pelican/contents.py

```diff
@@ -93,13 +93,16 @@ def __init__(self, content, metadata=None, settings=None,
         # create the slug if not existing, generate slug according to
         # setting of SLUG_ATTRIBUTE
         if not hasattr(self, 'slug'):
-            if settings['SLUGIFY_SOURCE'] == 'title' and hasattr(self, 'title'):
-                self.slug = slugify(self.title,
-                                settings.get('SLUG_SUBSTITUTIONS', ()))
-            elif settings['SLUGIFY_SOURCE'] == 'basename' and source_path != None:
-                basename = os.path.basename(os.path.splitext(source_path)[0])
-                self.slug = slugify(basename,
-                                settings.get('SLUG_SUBSTITUTIONS', ()))
+            if settings['SLUGIFY_SOURCE'] == 'title' and \
```

Comment:
sadly that is not possible because it would line up with the intendation of the next logical block (4 chars `if (`) which throws another error.

another solution for this is double intending the second and line which results in the following: